server/service/http/api.go

```diff
@@ -178,3 +179,27 @@ func (a *API) route(writer http.ResponseWriter, req *http.Request) {
 
 	a.respond(writer, string(resultByte))
 }
+
+type DeleteTableRequest struct {
+	ClusterName string `json:"clusterName"`
+	SchemaName  string `json:"schemaName"`
+	Table       string `json:"table"`
+}
+
+func (a *API) deleteTable(writer http.ResponseWriter, req *http.Request) {
+	var deleteTableRequest DeleteTableRequest
+	err := json.NewDecoder(req.Body).Decode(&deleteTableRequest)
+	if err != nil {
+		log.Error("decode request body failed", zap.Error(err))
+		a.respondError(writer, ErrParseRequest, "decode request body failed")
+		return
+	}
+
+	if err := a.clusterManager.DropTable(context.Background(), deleteTableRequest.ClusterName, deleteTableRequest.SchemaName, deleteTableRequest.Table); err != nil {
+		log.Error("cluster drop table failed", zap.Error(err))
+		a.respondError(writer, ErrParseResponse, "cluster drop table failed")
+		return
+	}
+
+	a.respond(writer, "ok")
```

Comment:
Please return msg in json format, not just 'ok'.

Reply:
Function `respond` return with struct `response`, it is defined as: 
```
type response struct {
	Status string      `json:"status"`
	Data   interface{} `json:"data,omitempty"`
	Error  string      `json:"error,omitempty"`
}
```
'ok' will be set in `response.Data`, and it is enough when the http interface does not need to return additional information.